### PR TITLE
Updated scripts for Container

### PR DESCRIPTION
**Pull request by mrrajan**
This PR includes changes for Containerization.

Signed-off-by: Rajan Ravi <rravi@redhat.com>

### CODE REVIEW

File: build/Dockerfile [hunk 1]
```diff
@@ -42,4 +41,4 @@ RUN useradd -r -u 1001 -g rhoda-ci-users rhoda-ci-runner
 RUN chgrp -R 1001 $HOME/rhoda-ci && chown -R 1001 $HOME/rhoda-ci && chmod -R 744 $HOME/rhoda-ci
 USER 1001
 
-ENTRYPOINT ["./run_robot_test.sh","-s","-x"]
+ENTRYPOINT ["/bin/sh","-c","./run_robot_test.sh -s -f $RUN_SCRIPT_ARGS -x"]
```
Comment (ikanias) [inline build/Dockerfile:44]: What has changed here? You changed the location of ./run_robot_test.sh?
And can you explain what is the difference here between '-x' and '-f'? Is '-f' is by force?

**Review by ikanias — DISMISSED** [review]
Just have some questions

File: build/Dockerfile (hunk 1 above)
Comment (mrrajan) [reply build/Dockerfile:44]: @ikanias for run_robot_test.sh "-f" used to share the test-variables.yaml in command line. To support that, variable $RUN_SCRIPT_ARGS is used.

**Review by ikanias — APPROVED** [review]
LGTM

**Pull request merged**